cmd/minishift/cmd/config/view.go

```diff
@@ -74,7 +75,9 @@ func configView() error {
 		if excluded {
 			continue
 		}
-
+		if configViewFormat != DefaultConfigViewFormat {
+			configViewFormat = fmt.Sprintln(configViewFormat)
```

Comment:
so why is this newline needed in the first place? If it is needed, it seems to make more sense to not add it to the default template either and then add it where needed regardless. This removed an extra if statement one wonders about.